variables.tf

```diff
@@ -332,3 +332,79 @@ variable "trusted_profile_roles" {
   description = "List of roles given to the trusted profile."
   default     = ["Administrator"]
 }
+
+########################################################################################################################
+# CBR Variables
+########################################################################################################################
+
+variable "cbr_prefix" {
+  type        = string
+  description = "String to use as the prefix for all CBR resources"
+  default     = "acct-infra-base"
+}
+
+variable "cbr_allow_cos_to_kms" {
+  type        = bool
+  description = "Set rule for COS to KMS, default is true"
+  default     = true
+}
+
+variable "cbr_allow_block_storage_to_kms" {
+  type        = bool
+  description = "Set rule for block storage to KMS, default is true"
+  default     = true
+}
+
+variable "cbr_allow_roks_to_kms" {
+  type        = bool
+  description = "Set rule for ROKS to KMS, default is true"
+  default     = true
+}
+
+variable "cbr_allow_icd_to_kms" {
+  type        = bool
+  description = "Set rule for ICD to KMS, default is true"
+  default     = true
+}
+
+variable "cbr_allow_event_streams_to_kms" {
+  type        = bool
+  description = "Set rule for Event Streams (Messagehub) to KMS, default is true"
+  default     = true
+}
+
+variable "cbr_allow_vpcs_to_container_registry" {
+  type        = bool
+  description = "Set rule for VPCs to container registry, default is true"
+  default     = true
+}
+
+variable "cbr_allow_vpcs_to_cos" {
+  type        = bool
+  description = "Set rule for VPCs to COS, default is true"
+  default     = true
+}
+
+variable "cbr_allow_at_to_cos" {
+  type        = bool
+  description = "Set rule for Activity Tracker to COS, default is true"
+  default     = true
+}
+
+variable "cbr_allow_iks_to_is" {
+  type        = bool
+  description = "Set rule for IKS to IS (VPC Infrastructure Services), default is true"
+  default     = true
+}
+
+variable "cbr_allow_is_to_cos" {
+  type        = bool
+  description = "Set rule for IS (VPC Infrastructure Services) to COS, default is true"
+  default     = true
+}
+
+variable "cbr_kms_service_targeted_by_prewired_rules" {
+  type        = string
+  description = "IBM Cloud offers two distinct Key Management Services (KMS): Key Protect and Hyper Protect Crypto Services (HPCS). This variable determines the specific KMS service to which the pre-configured rules will be applied. Use the value 'key-protect' to specify the Key Protect service, and 'hs-crypto' for the Hyper Protect Crypto Services (HPCS)."
+  default     = "hs-crypto"
```

Comment:
The CBR module has tis variable as a list. Is there a reason why you have it as a string? I guess there may be a use case where users may have Key Protect and HPCS, so thats probably why the CBR module has it as a list? (@Ak-sky is that right?)